pir/cpp/payload.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto2";
+
+package pir;
+
+message Payload {
```

Comment:
Payload is somewhat generic. Perhaps something like PIRResponse or Reply? Also: please add a comment describing the purpose of this message.